python/lsst/ip/diffim/subtractImages.py

```diff
@@ -297,6 +313,10 @@ def run(self, template, science, sources, finalizedPsfApCorrCatalog=None):
             convolveTemplate = False
         else:
             raise RuntimeError("Cannot handle AlardLuptonSubtract mode: %s", self.config.mode)
+        # put the template on the same photometric scale as the science image
+        photoCalib = template.getPhotoCalib()
+        self.log.info("Applying photometric calibration to template: %f", photoCalib.getCalibrationMean())
+        template.maskedImage = photoCalib.calibrateImage(template.maskedImage)
```

Comment:
Where is the science image involved here?

Reply:
It's not, the template is read in with units of counts, so this applies the template's calibration to put it in nJy. That should make it comparable to the science image, which should make work slightly easier for calculating the matching kernel.